src/pages/blog/airship-update-december-2020.md

```diff
@@ -0,0 +1,134 @@
+- [*Pre-Alpha Airship Blog Series 2 - An Educated Evolution*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-2-an-educated-evolution/)
+- [*Pre-Alpha Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-3-airship-2.0-architecture-high-level/)
+- [*Pre-Alpha Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-4-shipyard-an-evolution-of-the-front-door/)
+- [*Pre-Alpha Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-5-drydock-and-its-relationship-to-cluster-api/)
+- [*Pre-Alpha Airship Blog Series 6 - Armada Growing Pains*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-6-armada-growing-pains/)
+
+<br>
+
+Stay tuned for upcoming blog posts as Airship 2.0 progresses through release!
+
+<br>
+
+## **OPEN INFRASTRUCTURE SUMMIT OCTOBER 2020**
+
+Last month there were several excellent presentations related to Airship at the Open Infrastructure Summit. You can find
+each of these below:
+
+* [Airship 101](https://www.youtube.com/watch?v=Fh-YJDpQ5xE) (Andrew Karandjeff, Drew Walters)
+* [Own your YAML: extending Kustomize via Plugins](https://www.youtube.com/watch?v=Xoh_OpLoVtI) (Matt McEuen)
+* [Tackling security challenges with Airship](https://www.youtube.com/watch?v=9Qww2qHhNmE) (Alexander Hughes)
+* [Machine Learning at Edge Cloud](https://www.youtube.com/watch?v=TJlj8oMONPo) (Prakash Ramchandran, Vivek Hariharan)
+* [Significance of Hardware Classification combined with Host Configuration Operator](
+  https://www.youtube.com/watch?v=S0tJapmYNP4) (Digambar Patil, John Williams, Sirisha Gopigiri)
+* [China Mobile software and hardware integrated portable cloud platform test system](
+  https://www.youtube.com/watch?v=8N5PeW2l7Lc) (Xiaoguang Zhang, Zhiqiang Yu)
+* [hardware automation deployment and test with big scale in ChinaMobile NFV cloud](
+  https://www.youtube.com/watch?v=omKxP-qiKng) (Xiaoguang Zhang, Zhiqiang Yu)
+
+<br>
+
+## **AIRSHIP CONTAINER AS A SERVICE**
+
+Two operators will manage the lifecycle of Virtual Machines and their relationship to the cluster: vNode-Operator (ViNO)
+and the Support Infra Provider (SIP). These operators are new projects to the Airship ecosystem and were migrated to
+OpenDev earlier this month.
+
+So what do each of these operators do, at a thousand-foot view?
+
+[ViNO](https://opendev.org/airship/vino) (Virtual Node Operator)
+- Lays down libvirt and Virtual Machine definitions across bare-metal hosts
+- Fronts the Virtual Machines with sushy-tools for Redfish interactions via Metal3 et al.
+- Generates BareMetalHost definitions for the Virtual Machines
+
+<br>
+
+[SIP](https://opendev.org/airship/sip) (Support Infrastructure Provider)
+- Creates a load balancer
```

Comment:
Should we narrow in on the purpose of this load balancer here?  Can still keep it very short but I could see people wondering if this load balancer's intention is narrow in scope, or can be used broadly.

Reply:
done